src/templates/bootstrap/components/content.html

```diff
@@ -0,0 +1,5 @@
+<div [formGroup]="form" class="form-group">
```

Comment:
You do not need the form-group wrapper as well as the [formGroup] in this widget. There are not any input elements that would be inside this component.